runtime/src/main/java/org/corfudb/util/LambdaUtils.java

```diff
@@ -154,4 +157,18 @@ public static Object executeUnknownLambda(Object unknownLambda, Object... argume
     interface LambdaResolver<T> {
         Object resolve(T resolver, Object[] args);
     }
+
+    /**
+     * Suppresses all exceptions. This is used for scheduling tasks to ScheduledExecutorService.
+     * ScheduledExecutorService crashes in case a scheduled thread throws an Exception.
+     *
+     * @param runnable Task whose exceptions are to be suppressed.
+     */
+    public static void suppressExceptions(Runnable runnable) {
```

Comment:
suppress might be a strong word for this. I do not have good suggestions but something like runNoThrow might be more appropriate.